mmv1/products/vertexai/FeatureOnlineStore.yaml

```diff
@@ -103,3 +133,64 @@ properties:
               A percentage of the cluster's CPU capacity. Can be from 10% to 80%.
               When a cluster's CPU utilization exceeds the target that you have set,
               Bigtable immediately adds nodes to the cluster. When CPU utilization is substantially lower than the target, Bigtable removes nodes. If not set will default to 50%.
+  - !ruby/object:Api::Type::NestedObject
+    name: optimized
+    description:
+      Contains settings for the Optimized store that will be created to serve featureValues for all FeatureViews under this FeatureOnlineStore
+    conflicts:
+      - embeddingManagement
+    exactly_one_of:
```

Comment:
I believe you will need to update `bigtable` to include this same `exactly_one_of` block.

Reply:
Yeah, thanks for pointing it out. I had it originally but seems to have gotten removed during rebase. Added it back